perun-core/src/test/java/cz/metacentrum/perun/core/AbstractPerunIntegrationTest.java

```diff
@@ -50,7 +49,6 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = { "classpath:perun-base.xml", "classpath:perun-core.xml" })
-@Rollback
```

Comment:
Is now rollback feature default in ``@Transactional(transactionManager = "springTransactionManager")`` ? We usually used ``@TransactionConfiguration(transactionManager = "springTransactionManager", defaultRollback = true)`` in old code. Because transactional is generic annotation, does Spring behave different for "code" and "tests" ?